Firestore/core/include/firebase/firestore/firestore.h

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// TODO(rsgowman): This file isn't intended to be used just yet. It's just an
+// outline of what the API might eventually look like. Most of this was
+// shamelessly stolen and modified from rtdb's header file, melded with the
+// firestore api.
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_PUBLIC_FIRESTORE_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_PUBLIC_FIRESTORE_H_
+
+// TODO(rsgowman): replace these forward decl's with appropriate includes (once
+// they exist)
+namespace firebase {
+class App;
+class InitResult;
+}  // namespace firebase
+
+namespace absl {
```

Comment:
This exists: `#include "absl/strings/string_view.h"`.

Reply:
It turns out we don't use string_view anywhere in here, so I've just removed this altogether.